openhtf/io/frontend/__init__.py

```diff
@@ -170,9 +170,9 @@ def _discover(self):
           if (hostport, test_uid) not in self._watchers:
             self._watchers[hostport, test_uid] = TestWatcher(
                 hostport, remote_test,self._on_update_callback)
-      except (socket.error, station_api.StationUnreachableError) as e:
+      except station_api.StationUnreachableError:
         _LOG.debug('Station at %s is unreachable.', hostport)
-
+    
```

Comment:
@grybmadsci I didn't want to specify who was at fault ;). If we want to make a change to master, should we also make a similar change to station_api or will they merge soon (by next week)?

@jettisonjoe git/GitHub have a problem with it, take it up with Linus!